packages/shared/src/lib/func.ts

```diff
@@ -144,3 +151,13 @@ export const initReactModal = ({
 
   globalThis.reactModalInit = true;
 };
+
+export const initApp = (): void => {
+  const param = globalThis?.location?.search;
+  if (param?.includes('android=true')) {
```

Comment:
let's parse it as `URLSearchParams` and check with `get`, string compare can get it wrong.